contributing/code_of_conduct/reporting_guidelines.rst

```diff
@@ -0,0 +1,98 @@
+services first by calling the relevant local authorities. If you are unsure what
+service or agency is appropriate to contact, include this in your report and we
+will attempt to notify them.
+
+In your report please include:
+
+  * Your contact info for follow-up contact.
+  * Names (legal, nicknames, or pseudonyms) of any individuals involved.
+  * If there were other witnesses besides you, please try to include them as well.
+  * When and where the incident occurred. Please be as specific as possible.
+  * Your account of what occurred.
+  * If there is a publicly available record (e.g. a mailing list archive or a
+    public IRC or Slack log), please include a link.
+  * Any extra context you believe existed for the incident.
+  * If you believe this incident is ongoing.
+  * Any other information you believe we should have.
+
+What happens after you file a report?
+-------------------------------------
+
+You will receive a reply from the :doc:`enforcement team </contributing/code_of_conduct/enforcement_team>`
+acknowledging receipt as soon as possible, but within 24 hours.
+
+The team will immediately meet to review the incident and determine:
+
+  * What happened.
+  * Whether this event constitutes a code of conduct violation.
+  * What kind of response is appropriate.
+
+If this is determined to be an ongoing incident or a threat to physical safety,
+the team's immediate priority will be to protect everyone involved. This means
+we may delay an "official" response until we believe that the situation has ended
+and that everyone is physically safe.
+
+Once the team has a complete account of the events, they will make a decision as
+to how to respond. Responses may include:
+
+  * Nothing (if we determine no code of conduct violation occurred).
+  * A private reprimand from the Code of Conduct response team to the individual(s)
+    involved.
+  * A public reprimand.
+  * An imposed vacation (i.e. asking someone to "take a week off" from a mailing
+    list or Slack).
+  * A permanent or temporary ban from some or all Symfony conference/community
+    spaces (events, meetings, mailing lists, IRC, Slack, etc.)
+  * A request to engage in mediation and/or an accountability plan.
+  * On a case by case basis, the enforcement team might share information an incidents
+    with employers or other 3rd parties in order to fulfill their mandate.
+
+We'll respond within one week to the person who filed the report with either a
+resolution or an explanation of why the situation is not yet resolved.
+
+Once we've determined our final action, we'll contact the original reporter to
+let them know what action (if any) we'll be taking. We'll take into account feedback
+from the reporter on the appropriateness of our response, but our response will be
+determined by what will be best for community safety.
+
+The enforcement team keeps a private record of all incidents. By default all reports
```

Comment:
Are there any GDPR considerations for this? If this private record contains personally identifiable information, any processing would require consent from the people involved.

Reply:
We should really ask to some big open source project based in Europe which already has all this set up. I mean, we're not doing anything special, new or different. This CoC thing must be a "solved problem" in lots of other place. So let's copy them and move on 😄

Reply:
This is fine according to GDPR. I'll claim it is for a common interest where the community's interest is larger then the individual's. This information is not classed as "extra sensitive" so we are fine. 
Im also happy to let anybody challenge me in court over this. 

I think Javier's suggestion is good, we will look what others have done and move on.

Reply:
Well, no, we're not doing anything new, but the GDPR is new, so it's not unlikely that other projects might not have updated their CoCs to reflect this.

@Nyholm Are you sure about that? I presume you're referring to the grounds in 6.1f? From what I've read I'm not sure about that being applicable.

Reply:
On this topic, I hade a discussion with a co-worker of mine, who (while not a lawyer) is working with making their (large) business GDPR compliant. His best guess is that as long as everyone involved has *actively accepted* the Code of Conduct, that would be a legal basis for processing this data. That however would mean that we'd have to have some mechanism of enforcing that acceptance in order to participate in the project.

Reply:
Without more information, in my opinion that would be a big NO. Adding mandatory acceptance clauses in GitHub repos (as some companies do) creates a big friction and it feels very unwelcoming.

This could be solved as follows: in the web form used to report incidents, you add a big checkbox that the reporter must accept explicitly and that reads like this: *"We'll preserve your full anonimity ... blah blah ... but you let us store the details of this report ... blah blah ... we'll comply with the GDPR regulation ..."*

Reply:
The issue isn't so much storing information about the reporter, but rather the storing of information about the alleged infractor. While the reporter is likely to consent to storing data, the alleged infractor is not.

Reply:
I see. This is more complex than it looks :(